content/terraform/v1.13.x/docs/language/meta-arguments/count.mdx

```diff
@@ -0,0 +1,125 @@
+---
+page_title: count meta-argument reference
+description: Learn how the `count` meta-argument works in Terraform configuration.
+---
+
+# `count` reference
+
+By default, [`resource` block](/terraform/language/block/resource) configures one real infrastructure object. Similarly, a [`module` block](/terraform/language/block/module) includes a child module's contents into the configuration one time. 
```

Comment:
```suggestion
By default, a [`resource` block](/terraform/language/block/resource) configures one real infrastructure object. Similarly, a [`module` block](/terraform/language/block/module) includes a child module's contents into the configuration one time. 
```

Reply:
Should we also include the `ephemeral` block here?

Reply:
Actually, I see we explicitly list each supported block below, so probably fine to omit it here.